plugins/woocommerce/includes/class-wc-comments.php

```diff
@@ -215,6 +215,48 @@ public static function clear_transients( $post_id ) {
 		}
 	}
 
+	/**
+	 * Callback for 'wp_insert_comment' to delete the comment count cache if the comment is included in the count.
+	 *
+	 * @param int        $comment_id The comment ID.
+	 * @param WP_Comment $comment Comment object.
+	 *
+	 * @return void
+	 */
+	public static function delete_comments_count_cache_on_wp_insert_comment( $comment_id, $comment ) {
+		if ( ! self::is_comment_excluded_from_wp_comment_counts( $comment ) ) {
+			self::delete_comments_count_cache();
```

Comment:
Is it possible to adjust the cache count on insert if the comment is qualified for the count (similar to https://github.com/woocommerce/woocommerce/pull/54034)?  Or do we have to remove the cache and recalculate this on these occassions?